### ISSUE TITLE
Steam Inventory Interface

### ISSUE THREAD

**Issue by tylerhasman** [code-disaster/steamworks4j#92]
I would appreciate if the Steam Inventory Interface was implemented

**Comment by kevinmurphy678** [comment]
Yes please

**Comment by Zweistein2** [comment]
I'm also interested in the inventory interface

**Comment by Zweistein2** [comment]
I'm currently implementing the SteamInventory - interface. The first few tests were successful as I was able to create some items and add those to my inventory.

I'll create the pull-request the next few weeks

**Comment by Zweistein2** [comment]
The PR is created. Until it is reviewed by @code-disaster you can use the PR / my feature-branch

**Comment by kevinmurphy678** [comment]
> The PR is created. Until it is reviewed by @code-disaster you can use the PR / my feature-branch

Nice! I'll give it a try soon. Thanks for implementing it

**Comment by Zweistein2** [comment]
Doesn't seem like it will be merged soon...

**Comment by LukasS92** [comment]
Hey :) is it possible to use your implementation anyway?

**Comment by Zweistein2** [comment]
Yes, but you have to checkout the project, merge my branch and build the binaries by yourself locally.

**Comment by LukasS92** [comment]
Thank you, unfortunately I'm not that familiar with git to this extent. We need help with this task and of course would be happy to pay you for your time and effort. 
It would be great if you could contact me on Discord: Lukas#5909. We're from Germany too, if you like we can continue in German on Discord.
